applications/portfolioDrafts/jest.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    testEnvironment: 'node',
+    roots: ['<rootDir>/test'],
+    testMatch: ['**/*.test.ts'],
+    transform: {
+      '^.+\\.tsx?$': 'ts-jest'
+    },
+    collectCoverage: true,
```

Comment:
There is a function invoked createPortfolioStep.ts, `const command = portfolioStepCommand(TABLE_NAME, portfolioDraftId, portfolioStep);` and some others that are not being included in the code coverage report. 
```
collectCoverageFrom: [
"**/*.ts"
]
```
yields the same results --> only validtions.ts is tested.
`collectCoverageFrom=<glob>` uses `glob` which I don't have the most experience with. Docs here: https://jestjs.io/docs/cli#--collectcoveragefromglob

Reply:
This is now resolved.